group.py

```diff
@@ -2,4 +2,28 @@
 
 # Your code to go here...
 
-my_group =
+class person : 
+    def __init__(self, name, age, job=None):
+        self.name = name
+        self.age = age
+        self.job = job
+        self.relations = []
```

Comment:
You have expressed the relations as a _list of dictionaries_ here. This might get tricky if you we're trying to find a specific person in someone's relations as you'd have to know the index (a number) of that person in your list. If you expressed relations as a _dictionary of dictionaries_  you could search for the person by name instead.